src/web3-functions/event/index.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Web3Function,
+  Web3FunctionEventContext,
+} from "@gelatonetwork/web3-functions-sdk";
+
+Web3Function.onRun(async (context: Web3FunctionEventContext) => {
+  const { log } = context;
+
+  if (log.blockNumber !== 9727562) {
+    throw new Error("Log file fetched invalid");
+  }
```

Comment:
could you include a simple log parsing & decoding as this could serve for reference

Reply:
added in 4ef70ea620aff6abd690767799efb05e6e0b6f76